packages/react-datepicker.js

```diff
@@ -6008,7 +5973,7 @@ function getOffsetRectRelativeToArbitraryNode(children, parent) {
     offsets.marginLeft = marginLeft;
   }
 
-  if (isIE10 && !fixedPosition ? parent.contains(scrollParent) : parent === scrollParent && scrollParent.nodeName !== 'BODY') {
```

Comment:
What is the order of operations here? Is it `(isIE10 && !fixedPosition) ? ...` or `isIE10 && (!fixedPosition ? ...)`? I think that determines how to rewrite this

Reply:
[Precedence](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_precedence) is 4 for logical AND vs 2 for ternary operator

It is possible to put braces like this: 
`if ((isIE10 && !fixedPosition) ? parent.contains(scrollParent) : parent === scrollParent && scrollParent.nodeName !== 'BODY')`

As `isIE10` is always `false` we could substitute with
`if ((false && !fixedPosition) ? parent.contains(scrollParent) : parent === scrollParent && scrollParent.nodeName !== 'BODY')`

In this case `false && !fixedPosition` will always evaluate to `false`. No matter the value of `fixedPosition`. So the entire expression will evaluate to what is after colon